internal/admin/handlers.go

```diff
@@ -27,7 +27,7 @@ type Config struct {
 	// Password is an admin password.
 	Password string
 
-	// Secret is a secret to generate auth token for admin requests.
+	// TokenHMACSecretKey is a secret to generate auth token for admin requests.
```

Comment:
Looks like excess auto replacement

Reply:
Yeah that was the first thing I did. Im on the correct path now. I will fix,please ignore.